Core/Tests/CAN/CANTest.c

```diff
@@ -0,0 +1,10 @@
+#include <stdio.h>
+#include "../includes/DBCParser.h"
```

Comment:
This file cannot be found.

Reply:
This was never updated after organzing Inc folder, will fix